CHANGELOG.md

```diff
@@ -28,6 +28,9 @@ Please only add new entries below the [Unreleased](#unreleased---releasedate) he
 
 - fix broken links and format the example code (#526 @M-Adoo)
 
+### Features
+- Support the overlay (@wjian23)
```

Comment:
As a so important feature, we should provide a more detailed introduction to the overlay.

And I think we should write a chapter about it and put it in `docs/en/advanced_topics/overlay.md`.

Reply:
have put more example to rust doc.